src/ResourceManager/Compute/Commands.Compute/AvailabilitySets/GetAzureAvailabilitySetCommand.cs

```diff
@@ -48,7 +48,33 @@ public override void ExecuteCmdlet()
 
             ExecuteClientAction(() =>
             {
-                if (string.IsNullOrEmpty(this.Name))
+                if (string.IsNullOrEmpty(this.ResourceGroupName) && string.IsNullOrEmpty(this.Name))
```

Comment:
do we have a test case covering this new scenario?

Reply:
Updated AvailabilitySet test by adding this scenario.